### ISSUE TITLE
Feature request: charts

### ISSUE THREAD

**Issue by mixartemev** [maxkalik/CurRates#5]
What about historical prices chart for each currency pairs?

**Comment by maxkalik** [comment]
I had time just till this morning. But it's a great idea to implement historical chart.
I will try probably to integrate firstly the pull to refresh feature and then polling the server each 2 sec.

**Comment by mixartemev** [comment]
Polling?😒
Why not web-sockets?
(check out my [websoket powered chart with orderbook](https://hftrade.ru))

**Comment by maxkalik** [comment]
Ow! Great! Where I can get an api? I'd like to try :D
My task was simple: I had to use the particular endpoint.

**Comment by mixartemev** [comment]
The first websocket supporting api I found among free is [tiingo api](https://api.tiingo.com/documentation/websockets), but I use https://alor.dev as I'm an Alor broker client.

**Comment by maxkalik** [comment]
Thanks bro. I will try tiingo. When we create something useful together finally? :)

**Comment by mixartemev** [comment]
Yes! I want to make convenient semi-automatic trader client, and I need your swift and desing skills:)
